lib/eventlog/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
-
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/lib/eventlog/src/include)
 add_library(eventlog SHARED
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evtctx.c
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evtfmt.c
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evtout.c
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evtrec.c
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evtstr.c
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evtsyslog.c
-   ${CMAKE_CURRENT_SOURCE_DIR}/src/evttags.c
+  src/evtctx.c
+  src/evtfmt.c
+  src/evtout.c
+  src/evtrec.c
+  src/evtstr.c
+  src/evtsyslog.c
+  src/evttags.c
 )
 
+target_include_directories(eventlog SYSTEM INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/src)
```

Comment:
evtlog should not -isystem her own include directory.

Reply:
It's not its own include directory, its here for the users of evtlog (interface).

Reply:
Thanks for explanation!